test/arranger-conduit/AuthFunctions.t.sol

```diff
@@ -36,4 +36,32 @@ contract ArrangerConduit_AuthTests is ConduitTestBase {
         assertEq(conduitProxy.implementation(), newImplementation);
     }
 
+    function test_setBroker_noAuth() external {
+        vm.prank(makeAddr("non-admin"));
+        vm.expectRevert("ArrangerConduit/not-authorized");
+        conduit.setBroker(makeAddr("broker"), makeAddr("asset"), true);
+    }
+
+    function test_setBroker() external {
+        address asset1  = makeAddr("asset1");
+        address asset2  = makeAddr("asset2");
+        address broker1 = makeAddr("broker1");
+        address broker2 = makeAddr("broker2");
+
+        _assertBrokerSetter(broker1, asset1);
+        _assertBrokerSetter(broker1, asset2);
+        _assertBrokerSetter(broker2, asset1);
+        _assertBrokerSetter(broker2, asset2);
+    }
+
+    function _assertBrokerSetter(address broker, address asset) internal {
+        assertTrue(!conduit.isBroker(broker, asset));
+
+        conduit.setBroker(broker, asset, true);
+
+        assertTrue(conduit.isBroker(broker, asset));
+
+        conduit.setBroker(broker, asset, false);
```

Comment:
Maybe assert it's false after this call?

Reply:
Updated